Shared/parameters.json

```diff
@@ -48,6 +48,9 @@
         "keyVaultName": {
             "value": "__keyVaultName__"
         },
+        "keyVaultResourceGroup": {
```

Comment:
All of the new parameters need to move into the shared parameters and shared template too otherwise the full infrastructure deployment wont work.